app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -116,6 +120,9 @@ protected ListPopup buildPopupMenu(Context context, ArrayAdapter<ListPopup.Item>
         }
         adapter.add(new ListPopup.Item(context, R.string.menu_exclude));
         adapter.add(new ListPopup.Item(context, R.string.menu_favorites_add));
+        adapter.add(new ListPopup.Item(context, R.string.menu_tags_edit));
+        adapter.add(new ListPopup.Item(context, R.string.menu_app_rename));
+        adapter.add(new ListPopup.Item(context, R.string.menu_custom_icon));
```

Comment:
I think we should only display this option if we're using a custom icon pack, as it is not useful otherwise